lbrynet/dht/node.py

```diff
@@ -475,7 +475,7 @@ def store(self, key, value, originalPublisherID=None, self_store=False, **kwargs
 
         if 'port' in value:
             port = int(value['port'])
-            if 0 <= port <= 65536:
+            if 0 <= port <= 65535:
```

Comment:
is 0 a valid port?

Reply:
potentially yes, but it's reserved and can't really be used, so we could also exclude it from the check

Reply:
the check has been changed